README.md

```diff
@@ -1,18 +1,18 @@
 # Cass Operator
-[![Gitter](https://badges.gitter.im/cass-operator/community.svg)](https://gitter.im/cass-operator/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-[![Go Report Card](https://goreportcard.com/badge/github.com/datastax/cass-operator)](https://goreportcard.com/report/github.com/datastax/cass-operator)
-[![License: Apache License 2.0](https://img.shields.io/github/license/datastax/cass-operator)](https://github.com/datastax/cass-operator/blob/master/LICENSE.txt)
+[![License: Apache License 2.0](https://img.shields.io/github/license/k8ssandra/cass-operator)](https://github.com/k8ssandra/cass-operator/blob/master/LICENSE.txt)
 
-The DataStax Kubernetes Operator for Apache Cassandra&reg;
+The k8ssandra Kubernetes Operator for Apache Cassandra&reg;. This repository replaces the old [datastax/cass-operator](https://github.com/datastax/cass-operator) for use-cases in the k8ssandra project. Some documentation is still out of date and will be modified in the future. Check [k8ssandra/k8ssandra](https://github.com/k8ssandra/k8ssandra) for more up to date information.
 
 ## Getting Started
 
+To create a full featured cluster, the recommend approach is to use the Helm charts. Check the [Getting started](https://k8ssandra.io/docs/getting-started/) documentation at (k8ssandra.io)[https://k8ssandra.io/docs].
```

Comment:
Should this go to a dedicated page covering how to just install the `cass-operator` helm chart?

For context, @johnsmartco has a section of the new docs site which will be dedicated to cass-operator.